cognite/neat/_graph/loaders/_rdf2dms.py

```diff
@@ -584,13 +587,20 @@ def _upload_to_cdf(
                 skip_on_version_conflict=True,
             )
         except CogniteAPIError as e:
-            result = UploadResult[InstanceId](name=name, issues=read_issues)
-            result.error_messages.append(str(e))
-            result.failed_upserted.update(item.as_id() for item in e.failed + e.unknown)
-            result.created.update(item.as_id() for item in e.successful)
-            yield result
+            if len(items) == 1:
+                yield UploadResult(
+                    name=name,
+                    issues=read_issues,
+                    failed_items=items,
+                    error_messages=[str(e)],
+                    failed_upserted={item.as_id() for item in items},  # type: ignore[attr-defined]
+                )
+            else:
+                half = len(items) // 2
+                yield from self._upload_to_cdf(client, items[:half], dry_run, read_issues, class_name)
+                yield from self._upload_to_cdf(client, items[half:], dry_run, read_issues, class_name)
```

Comment:
Binary search for problematic items. Storing the failed items for easier debugging.